package.json

```diff
@@ -29,18 +29,14 @@
   "dependencies": {
     "ansi-256-colors": "^1.1.0",
     "chalk": "^2.0.1",
-    "chokidar": "^2.1.2",
     "debug": "^4.1.1",
-    "fs-extra": "^7.0.1",
-    "generator-nuxeo": "github:nuxeo/generator-nuxeo#master",
+    "generator-nuxeo": "github:nuxeo/generator-nuxeo#improvement-NXCLI-55-allow-sync-with-docker",
```

Comment:
Keep `#master` commited

Reply:
Yes, I know, it was just to build the devcontainer images with the changes done in the PR